spk/medusa/Makefile

```diff
@@ -0,0 +1,39 @@
+SPK_NAME = medusa
+SPK_VERS = $(shell date +%Y%m%d)
+SPK_REV = 2
+SPK_ICON = src/medusa.png
+
+SPK_DEPENDS = "python3>=3.5.6-8:git"
+
+MAINTAINER = SynoCommunity
+DESCRIPTION = Automatic Video Library Manager for TV Shows. It watches for new episodes of your favorite shows, and when they are posted it does its magic.
+RELOAD_UI = yes
+STARTABLE = yes
+DISPLAY_NAME = Medusa
+CHANGELOG = "Separate service for Medusa"
+
+HOMEPAGE   = http://pymedusa.com
+LICENSE    = GPL
+
+SERVICE_USER = auto
+SERVICE_SETUP = src/service-setup.sh
+SERVICE_PORT = 8081
```

Comment:
this port is already used by synocommunity package sickbeard.
see wiki page https://github.com/SynoCommunity/spksrc/wiki/SynoCommunity-Used-Ports

please choose a different port, adjust the installation/configuration and document the port on the wiki page.

OR: define SPK_CONFLICT = sickbeard when parallel installations are not possible due concurrent functions (not concurrent ports alone).